Formula/xinit.rb

```diff
@@ -0,0 +1,152 @@
+class Xinit < Formula
+  desc "Start the X Window System server"
+  homepage "https://gitlab.freedesktop.org/xorg/app/xinit"
+  url "https://www.x.org/releases/individual/app/xinit-1.4.1.tar.bz2"
+  sha256 "de9b8f617b68a70f6caf87da01fcf0ebd2b75690cdcba9c921d0ef54fa54abb9"
+  license all_of: ["MIT", "APSL-2.0"]
+
+  depends_on "pkg-config" => :build
+  depends_on "tradcpp"    => :build
+
+  depends_on "libx11"
+  depends_on "lndir"
+  depends_on "mkfontscale"
+  depends_on "xauth"
+  depends_on "xmodmap"
+  depends_on "xrdb"
+  depends_on "xserver"
+  depends_on "xterm"
+
+  on_macos do
+    depends_on "quartz-wm"
```

Comment:
Should we let user choose the windows manager, e.g. reuse the `quartz-wm` from `XQuartz`? 
Should we use `twm` on linux?